backend/app/tests/crud/collections/collection/test_crud_collection_read_all.py

```
@@ -30,15 +29,15 @@ def create_collections(db: Session, n: int):
 
 @pytest.fixture(scope="class")
 def refresh(self, db: Session):
```

Ruff (0.14.8)

Unused function argument: `self`

(ARG001)